checkov/cdk/checks/python/APIGatewayV2AccessLogging.yaml

```diff
@@ -0,0 +1,18 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_95
+  name: Ensure API Gateway V2 has Access Logging enabled
+  category: LOGGING
+scope:
+  languages:
+    - python
+definition:
+  patterns:
+    or:
+      - pattern: aws_cdk.aws_apigatewayv2.CfnStage(<ANY>)
+        conditions:
+          - not_pattern: aws_cdk.aws_apigatewayv2.CfnStage(<ANY>, access_log_settings=aws_cdk.aws_apigatewayv2.CfnStage.AccessLogSettingsProperty(<ANY>, destination_arn=$ARG ,<ANY>) ,<ANY>)
+      - pattern: aws_cdk.aws_apigatewayv2.CfnApi(<ANY>)
+        conditions:
+          - not_pattern: aws_cdk.aws_apigatewayv2.CfnApi(<ANY>, access_log_settings=aws_cdk.aws_apigatewayv2.CfnApi.AccessLogSettingsProperty(<ANY>, destination_arn=$ARG ,<ANY>) ,<ANY>)
```

Comment:
Seems to me they are the patterns are same & you can differ by $FUNC between them